### DISCUSSION TITLE
deno js runtime in domcloud

### DISCUSSION THREAD

**Discussion by elycheikhsmail** [domcloud/domcloud#41] [Ideas]
beside node it will usefull to have deno js runtime installed by default on the server

**Comment by willnode** [comment]
Deno setup now can be simplified through [the system runner](https://domcloud.co/docs/features/runner/#deno):

```yaml
features:
- deno
```

**Comment by elycheikhsmail** [comment]
thank you but where we can write recipe contents (or YAML file ) ?
please and deno runtime in the doc

**Reply by willnode** [reply]
Added some more explanation here: https://domcloud.co/docs/features/runner

**Comment by elycheikhsmail** [comment]
I read all the doc befor I ask, and read this specific page.
the YAML will be written in my code ? or in some input in domcloud dashboad ? I'm confusing there !

**Reply by willnode** [reply]
write the YAML to the system runner, a.k.a. the `Deploy` tab in DOM Cloud dashboard.

![image](https://user-images.githubusercontent.com/20214420/231613052-a3fb5c1b-cd2f-41fd-b192-4fd158ab8da7.png)

**Comment by elycheikhsmail** [comment]
thank you, I think this should be the place but I was not sure
